src/options.rs

```diff
@@ -408,7 +424,7 @@ impl Options {
             } else {
                 OutputMode::JsonElements
             },
-            no_version_check: tmp.no_version_check,
+            no_version_check: no_version_check,
```

Comment:
nit:

```suggestion
            no_version_check,
```

would be even better if we rename the variable and field name too for consistency.

Reply:
Updated!